src/main/java/gift/exception/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,27 @@
+package gift.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@RestControllerAdvice
+public class GlobalExceptionHandler {
+
+    @ExceptionHandler(MethodArgumentNotValidException.class)
+    public ResponseEntity<String> handleValidationExceptions(MethodArgumentNotValidException ex) {
+        String errorMessage = ex.getBindingResult()
+                .getFieldErrors()
+                .stream()
+                .map(error -> error.getDefaultMessage())
```

Comment:
`메서드 참조`로 변경해볼 수도 있을 것 같아요. 
```suggestion
                .map(DefaultMessageSourceResolvable::getDefaultMessage)
```